src/bindings/python/src/compatibility/openvino/inference_engine/ie_api.pyx

```diff
@@ -541,10 +541,10 @@ cdef class IECore:
 
 
     def register_plugin(self, plugin_name: str, device_name: str = ""):
```

Comment:
@ilyachur @ilya-lavrenov I decided to leave argument name as is because it may be used as `register_plugin(plugin_name=...)`. We may also change it to achieve full compatibility with other APIs, but it will break Py compatibility